src/features/offer/components/OfferBody/OfferBody.tsx

```diff
@@ -232,15 +239,28 @@ export const OfferBody: FunctionComponent<Props> = ({
       </SectionWithDivider>
 
       <SectionWithDivider visible={!offer.isDigital} margin>
-        <WhereSection
-          beforeNavigateToItinerary={() =>
-            analytics.logConsultItinerary({ offerId: offer.id, from: 'offer' })
-          }
-          venue={venue}
-          address={fullAddress}
-          locationCoordinates={venue.coordinates}
-          showVenueBanner={showVenueBanner}
-        />
+        {enableMultivenueOffer ? (
+          <VenueSection
+            beforeNavigateToItinerary={() =>
+              analytics.logConsultItinerary({ offerId: offer.id, from: 'offer' })
+            }
+            venue={venue}
+            locationCoordinates={venue.coordinates}
+            showVenueBanner={showVenueBanner}
+            title={getVenueSectionTitle()}
```

Comment:
Tu ne devrais pas rappeler la fonction `getVenueSectionTitle` à chaque render.
Par contre tu peux avoir un truc du genre
```ts
const venueSectionTitle = useMemo(() => ..., [])
```

et appeler
```suggestion
            title={venueSectionTitle}
```

Reply:
Impossible d'utiliser un useMemo parce qu'il y a la condition 
`if (!offer) return <React.Fragment></React.Fragment>`

Reply:
Ok je comprends. Je n'aime pas avoir un return en plein milieu justement pour cette raison.
Pour moi on devrait avoir 
```tsx
const Component = () => {
  useEffect(() => ...)
  const handler = useCallback(() => ...)

  if (!var) return
  return (...)
}
```

Ce qui impliquerait de faire des checks dans toutes les variables plus haut mais c'est mieux ainsi...
Enfin en ce qui te concerne OK, ça marche 👍